content/_index.html

```diff
@@ -44,23 +44,23 @@ <h2><i class="fa-light fa-chart-network"></i> Open Source Ecosystem </h2>
           <div class="column is-one-third">
             <h2><i class="fa-thin fa-objects-column"></i> Faster Pandas <br> with cuDF</h2>
             <p>cuDF is a near drop in replacement to pandas for most use cases and has greatly improved performance. </p> <br>
-            <a href="https://github.com/rapidsai/cudf/tree/branch-23.04/docs/cudf/source/user_guide/performance_comparisons.ipynb"  target="_blank">Run this benchmark yourself <i class="fa-solid fa-arrow-up-right"></i> </a>
+            <a href="https://github.com/rapidsai/cudf/blob/branch-23.06/docs/cudf/source/user_guide/performance-comparisons/performance-comparisons.ipynb"  target="_blank">Run this benchmark yourself <i class="fa-solid fa-arrow-up-right"></i> </a>
```

Comment:
I'm okay with updating the links -  we technically haven't tried the notebooks with the latest version. The fine print points that out, however.